pubspec.yaml

```diff
@@ -21,59 +21,59 @@ environment:
   sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  auto_route: ^3.2.0
-  bloc: ^8.0.1
+  auto_route: ^3.2.2
+  bloc: ^8.0.2
   cached_network_image: ^3.2.0
   country_codes: ^2.1.1
   country_icons: ^2.0.2
-  cupertino_icons: ^1.0.2
-  dartz: ^0.10.0-nullsafety.2
+  cupertino_icons: ^1.0.4
+  dartz: ^0.10.1
   dots_indicator: ^2.0.0
   email_validator: ^2.0.1
   equatable: ^2.0.3
-  expandable_page_view: ^1.0.9
-  firebase_analytics: ^9.0.5
-  firebase_auth: ^3.3.5
-  firebase_core: ^1.11.0
-  firebase_crashlytics: ^2.4.5
-  firebase_remote_config: ^1.0.4
+  expandable_page_view: ^1.0.10
+  firebase_analytics: ^9.1.0
+  firebase_auth: ^3.3.6
+  firebase_core: ^1.12.0
+  firebase_crashlytics: ^2.5.1
+  firebase_remote_config: ^2.0.0
   flutter:
     sdk: flutter
-  flutter_bloc: ^8.0.0
+  flutter_bloc: ^8.0.1
   flutter_dotenv: ^5.0.2
-  freezed_annotation: ^0.14.2
-  get_it: ^7.1.4
+  freezed_annotation: ^0.15.0
+  get_it: ^7.2.0
   http: ^0.13.4
-  image: ^3.1.0
+  image: ^3.1.1
   image_cropper: ^1.4.1
-  image_picker: ^0.8.4+4
-  injectable: ^1.4.1
+  image_picker: ^0.8.4+7
+  injectable: ^1.5.3
   intl: ^0.17.0
   json_annotation: ^4.4.0
   meta: ^1.7.0
   mocktail: ^0.2.0
-  package_info_plus: ^1.0.6
-  phone_number: ^0.11.0+2
-  rive: 0.7.28
-  rxdart: ^0.27.1
-  share_plus: ^2.1.4
-  shared_preferences: ^2.0.7
+  package_info_plus: ^1.3.0
+  phone_number: ^0.12.0+1
+  rive: 0.7.28 # Why is this a specific version?
```

Comment:
I still have this question.. 😂

Reply:
Tests seemed to fail if we used any above that. Not sure if it was fixed in the latest versions